Sources/TuistKit/Commands/RunCommand.swift

```diff
@@ -0,0 +1,25 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+
+struct RunCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "run",
+                             abstract: "Builds and runs a project")
+    }
+
+    @Argument(
+        help: "The scheme to be guild. By default it builds all the buildable schemes of the project in the current directory"
+    )
+    var schemeName: String
```

Comment:
If I'm not wrong, I think the `@Argument` property wrapper will infer the following argument names: `-s` & `--scheme-name`. Although the first one I think is right, I think the second one could be shorter and more aligned with `xcodebuild`: `--scheme`. You can achieve that by either renaming `schemeName` to `scheme`, or passing the `name` argument to the initialization of the property wrapper:

```swift
@Argument(name: [.customShort("s"), .customLong("scheme")])
```